src/Raven.Server/ServerWide/BackgroundTasks/LatestVersionCheck.cs

```diff
@@ -7,61 +8,69 @@
 using Raven.Server.Json;
 using Raven.Server.NotificationCenter.Notifications;
 using Raven.Server.NotificationCenter.Notifications.Details;
+using Sparrow.Collections;
 using Sparrow.Json;
 using Sparrow.Json.Parsing;
 
 namespace Raven.Server.ServerWide.BackgroundTasks
 {
-    public class LatestVersionCheck : IDisposable
+    public static class LatestVersionCheck
     {
         private const string ApiRavenDbNet = "https://api.ravendb.net";
 
-        private static readonly Logger _logger = LoggingSource.Instance.GetLogger<LatestVersionCheck>(null);
+        private static readonly Logger _logger = LoggingSource.Instance.GetLogger<ServerStore>(null);
```

Comment:
Why not use the LatestVersionCheck here? 
And the name shouldn't be null, can be "server" or "global" or something.